assets/js/components/layout/layout.js

```diff
@@ -62,15 +62,15 @@ class Layout extends Component {
 				{ header &&
 					<LayoutHeader
 						title={ title }
-						ctaLabel={ headerCtaLabel }
-						ctaLink={ headerCtaLink }
+						CTALabel={ headerCTALabel }
+						CTALink={ headerCTALink }
```

Comment:
The prop names for `LayoutHeader` shouldn't change here because the acronym capitalization only affects the letters/word when it falls in the middle of a variable where the first character would already be capitalized per normal camel case style.

Any variables that _start_ with, or are an acronym by themselves should remain lowercase. E.g. `url` or `ampMode` are correct forms of these variables containing acronyms.

Also, it looks like the case wasn't changed in the `LayoutHeader` and `LayoutFooter` components to match, but again – the prop names shouldn't change, just making an observation here.

Reply:
This is what happens when you just blindly do a search and replace 🤦 .
Thanks for catching that @aaemnnosttv . This has been updated